src/intrinsics/prepack/global.js

```diff
@@ -155,6 +155,39 @@ export default function(realm: Realm): void {
     configurable: true,
   });
 
+  if (realm.react.enabled) {
+    global.$DefineOwnProperty("__reactComponentRoots", {
```

Comment:
@cblappert This is unrelated to this PR. I saw that this pattern was added in the additional functions one, but why do we make these a public object instead of just an internal Map? Seems weird for the compiled script to be able to reason about this and mutate this.